tests/test_pedigree.py

```diff
@@ -362,6 +362,124 @@ def test_valid_pedigree(self):
         tc.tree_sequence()  # creating tree sequence should succeed
 
 
+class TestSimulateThroughPedigree:
+    def verify(self, input_tables):
+        initial_state = input_tables.tree_sequence()
+        ts = msprime.sim_ancestry(
+            model="wf_ped",
+            initial_state=initial_state,
+            recombination_rate=0,
```

Comment:
We should allow some recombination to make this interesting I think. How about ``sequence_length=100``, ``recombination_rate=0.5`` or something? So, we should have a good bit of recombination then.